src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I think that brings us back to the problem we were trying to avoid, which is that the "donor" event gets affected. That is why we wanted either a different empty event or no `originalEvent`.